src/core_plugins/kibana/public/home/components/recently_accessed.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { RecentlyAccessed, NUM_LONG_LINKS } from './recently_accessed';
+
+const createRecentlyAccessed = (length) => {
+  const recentlyAccessed = [];
+  while(recentlyAccessed.length < length) {
+    recentlyAccessed.push({
+      label: `label${recentlyAccessed.length}`,
+      link: `link${recentlyAccessed.length}`
+    });
+  }
+  return recentlyAccessed;
+};
+
+test('should render dropdown when recently accessed list size exceeds NUM_LONG_LINKS', () => {
+  const component = shallow(<RecentlyAccessed
```

Comment:
The snapshots for these tests are massive. How do you feel about using `findTestSubject` to find the specific elements you're looking for instead of snapshots?